test/static_assertions/plugin_no_base_class.cc

```diff
@@ -33,7 +33,7 @@ class NotBase
   int test3;
 };
 
-IGN_COMMON_REGISTER_SINGLE_PLUGIN(A, NotBase)
+GZ_COMMON_REGISTER_SINGLE_PLUGIN(A, NotBase)
```

Comment:
This can stay as it was, it's deprecated and we're not migrating it

Reply:
Fixed!